packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -431,18 +425,14 @@ describe('merge-on-green', () => {
         getMogLabel([{name: 'automerge'}]),
         getStatusi('6dcb09b5b57875f334f61aebed695e2e4193db5e', []),
         getRuns('6dcb09b5b57875f334f61aebed695e2e4193db5e', {
-          check_runs: [
-            {
-              name: 'Special Check',
-              conclusion: 'success',
-            },
-          ],
+          name: 'Special Check',
+          conclusion: 'success',
         }),
         getCommentsOnPr([]),
       ];
 
       await probot.receive({
-        name: 'schedule.repository',
+        name: 'schedule.repository' as any,
```

Comment:
we might want to link to this:

https://github.com/octokit/webhooks.js/issues/277

to show that we're moving away from this any logic.